types/musickit-js/MusicKit.d.ts

```diff
@@ -42,6 +42,10 @@ declare namespace MusicKit {
 
     const version: string;
 
+    interface AuthStatus {
+        AUTHORIZED: 3;
+    }
+
```

Comment:
These status codes can be seen on the Swift documentation: https://developer.apple.com/documentation/musickit/musicauthorization/status

![image](https://user-images.githubusercontent.com/81154791/147941908-6cdbfcb1-87bc-4a78-814f-ebe5fd712c1f.png)

From the minified output of the SDK, we can also infer the codes for the other statuses:

![image](https://user-images.githubusercontent.com/81154791/147942046-dad50413-1962-40f6-b36e-27b07887065d.png)

Therefore, we might want to extend the interface as follows (to include all available statuses). I didn't include `UNAVAILABLE: -1` because it's not documented on the Swift page, so it might be an internal status, but it's only a guess. Feel free to include it if you consider it helpful

```ts
    interface AuthStatus {
        NOT_DETERMINED: 0;
        DENIED: 1;
        RESTRICTED: 2;
        AUTHORIZED: 3;
    }
```